.github/ISSUE_TEMPLATE/story.md

```diff
@@ -0,0 +1,26 @@
+---
+name: Story
+about: A single work item, usually a part of a larger Epic.
+title: '[adds/updates/fixes]: [what] to/in [where]'
+---
+<!-- Provide a general summary of the issue in the title above -->
+
+```[TaskList]
+## Tasks
+- [ ] Peer review code - _should be done by another engineer in addition to the normal PR review process_
```

Comment:
This would be enforced by CODEOWNERS policies so a peer review task should be used for getting teammates eyes on each others code. It helps build a mentality of working within a team and also enables teammates to share their strengths with each other.

An example would be I could go implement a new FAST component for something and use `document.getElementById` to get references to elements in the component. While that works and is technically correct, you as a peer could review the PR and suggest replacing `document.getElementById` with FAST's `ref` directive which would be more succinct.

So each PR should have signoff from a code owner and another engineer. This process has a number of benefits, namely:
1. Encourages working as a team and preventing silos.
2. Enables teammates to share their experience with each other, improving the overall quality of each change to the codebase.
3. Helps free up leads and code owners time a bit more so that they don't get stuck in review hell.